challenges/week_1/bus_fare_challenge.py

```diff
@@ -1 +1,39 @@
 # WRITE YOUR CODE SOLUTION HERE
+from datetime import datetime, timedelta, date
+
+#Get todays date and store it in a variable 'date'
+
+date = datetime.now()
```

Comment:
This is almost correct, only that `datetime.now()` alone returns the full date including time and timezone. The challenge only needs the date, find a way to pull this off.
 **Hint:** *add another attribute to this line to specify the date alone.*